src/test/java/net/pms/test/formats/FormatFactoryTest.java

```diff
@@ -165,6 +165,7 @@ public final void testFormatRetrieval() {
 		testSingleFormat("test.wmv", "MPG", Format.VIDEO);
 		testSingleFormat("test.wtv", "MPG", Format.VIDEO);
 		testSingleFormat("test.mpa", "MPGAudio", Format.AUDIO);
+		testSingleFormat("test.mp1", "MPGAudio", Format.AUDIO);
```

Comment:
@Sami32 Go ahead, since most of the others are cryptic and reader-unfriendly, why should this be any better?